release.sh

```diff
@@ -2,32 +2,63 @@
 
 set -e
 
-last_tag=$(git tag --sort=-creatordate | head -n 1)
+last_tag=$(git tag | head -n 1)
 new_tag=$(semver bump patch "$last_tag")
 timestamp=$(date -u +%Y%m%d%H%M%S)
 ref=$(echo "$BRANCH" | sed -e "s/[^a-zA-Z0-9]//g")
 new_version=$(if [ "$ref" = "master" ]; then echo "${new_tag}"; else echo "${new_tag}-${ref}.$timestamp"; fi)
 
-echo "Bump version: ${last_tag} -> ${new_version}"
-git tag "${new_version}"
+validateFileChanges() {
+  #Define sensible file
+  sensible_files=(Dockerfile .dockerignore requirements.txt)
 
-# Create image tag
-image_tag="${IMAGE_NAME}:${new_version}"
-docker tag "${IMAGE_NAME}" "${image_tag}"
+  #Validate if sensible file are in list
```

Comment:
One space after the delimiter for cleanliness :). 
```suggestion
  # Validate if sensible file are in list
```